### PR TITLE
Make a production  docker image #12

### PR DESCRIPTION
**Pull request by georgettica**
This is supposed to start fixing #12 . please tell me what else needs to be done.

this might solve some other issues.

also, if this repo is stale I would like to know =]

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,32 @@
...
 RUN apk add --no-cache  git
 
+ENV BUILD_PATH /build/main
+
 COPY . /go/src/github.com/tmc/pqstream
 
 RUN go get -v github.com/tmc/pqstream/cmd/pqs \
     && go get -v github.com/tmc/pqstream/cmd/pqsd
 
+RUN mkdir -p $B{UILD_PATH}
```
Comment (tmc) [inline Dockerfile]: Looks like this missed the B

**Review by tmc — COMMENTED** [review]
Thanks for your contribution. Please see my comment.

File: Dockerfile (hunk 1 above)
Comment (georgettica) [reply Dockerfile]: Thanks!
fixed

**Comment by georgettica** [comment]
I have an issue with the code generation.. please tell me what I need to do to fix it.

this blocks the build-and-test check

**Comment by tmc** [comment]
Don’t worry about the check generated code. I need to update that. There’s been some drift. Can you update the title here to be more descriptive?

**Comment by georgettica** [comment]
Sure! I'll do that now

**Review by georgettica — COMMENTED** [review]
Done @tmc, please tell me it's ready

**Comment by georgettica** [comment]
Any progress @tmc ?

**Comment by tmc** [comment]
Thanks for your contribution. Opening an issue to address that drift.

**Pull request merged**